site/config.toml

```diff
@@ -25,6 +25,10 @@ disqusShortname = "checkly"
     name = "Guides"
     url = "/guides"
     weight = 20
+[[menu.topnav]]
+    name = "Learn"
+    url = "/learn"
```

Comment:
Could we link to `/learn/headless/` to avoid a redirect?

We also have always to remember the trailing slash `/`. Without it we cause google search console redirect errors.

Reply:
Fixed, added the trailing slash on learn and guides.